src/main/ipcs/setup.ipcs.ts

```diff
@@ -0,0 +1,11 @@
+import { from } from "rxjs";
+
+import { IpcService } from "main/services/ipc.service";
+import { SongDetailsCacheService } from "main/services/additional-content/maps/song-details-cache.service";
+
+const ipc = IpcService.getInstance();
+
+ipc.on("setup-done", (_, reply) => {
```

Comment:
Why do we need to load the `songs details cache` here ? since it do not rely on the installation path ?

Reply:
`SongDetailsCacheService` requires `ConfigurationService`. The configuration service needs the `config.cfg` file from the installation path in order for it to read the **etag** value.

![image](https://github.com/user-attachments/assets/cca8ca96-1ee9-41c6-9f94-aafdb98c9bd9)

Reply:
Oh yeah you're right i was forgetting the etagKey 👌